serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/mocking-opnapi-services-with-wiremock.adoc

```diff
@@ -11,31 +11,34 @@ Mocking OpenAPI services with WireMock
 :create-your-first-workflow-service: xref:getting-started/create-your-first-workflow-service.adoc
 :orchestration-of-openapi-based-services: xref:service-orchestration/orchestration-of-openapi-based-services.adoc
 
-This document describes how to mock OpenAPI services using WireMock. The testing procedure in this document is based on the `serverless-workflow-examples/serverless-workflow-service-calls-quarkus` example application. You can access this example application in the link:{kogito_sw_examples_url}/serverless-workflow-service-calls-quarkus[GitHub repository].
-
-== Prerequisites
-
-The examples in this document assumes that you have the following prerequisites:
-
-* You have installed the {getting-familiar-with-our-tooling}[Kogito tooling].
+This document describes how to mock OpenAPI services using WireMock. The testing procedure described in this document is based on the link:{kogito_sw_examples_url}/serverless-workflow-service-calls-quarkus[`serverless-workflow-service-calls-quarkus`] example application.
 
-* You have {create-your-first-workflow-service}[created a Serverless Workflow project] that {orchestration-of-openapi-based-services}[orchestrates OpenAPI services].
+link:{wiremock_url}[WireMock] is an open source Mocking API for unit, integration, and performance tests. You can use WireMock to isolate your tests from third party APIs and prototype APIs that do not exist.
 
-== About WireMock
+.Prerequisites
```

Comment:
We use "==" for the second-level heading. Since Prerequisites are not a dedicated section, therefore I would use it as is. 
Regarding the rendering part, we can make that top-level label with Antora. 

cc @ricardozanini

Reply:
Not in the top level section.

![image](https://user-images.githubusercontent.com/11776454/179234749-7beab7fc-6ce8-4afc-8716-722af6c6d354.png)

Reply:
@akumar074 can you take a look, please? I think the HTML/CSS elements, in this case, might be slightly different and it's not using our custom style.